app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<header class="login-header">
+<%= link_to root_path do %>
+<i class=" fa fa-arrow-left"></i>
+<% end %>
+<div class="login-link">
+<p>REGISTER</p>
+</div>
+<p class="p-hide">Next</p>
+</header>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+<div class="cont-top">
+  <div class="signup-top">
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, I think it would be better to follow the given design in your application layout and the distance between elements, Kindly try to replicate the template as much as possible :pray:

**Your design** | **Template**
| --- | --- |
<img width="401" alt="Screen Shot 2023-10-19 at 15 04 27" src="https://github.com/hmunish/ror-capstone/assets/102757126/f506cad9-04ea-4870-b07f-7faa7115b4ec"> | ![image](https://github.com/hmunish/ror-capstone/assets/102757126/c984c772-a326-4ed5-885a-552670dc18bf)

Reply:
@TuyishimireEric